### PR TITLE
feat: improved plugin #1631

### LINKED ISSUE #1631: improve Plugin & Plugin Editor & Meta Raw Data Editor module
Here are some minor issues on the Plugin & Plugin Editor & Meta Raw Data Editor page we could resolve.

## Plugin TODO

- [ ] Search area, using Table's own, using fuzzy search.
- [ ] page title not internationalized: configure global plugins/Configure Global Plugin.
- [ ] Add icon for Plugin Marketplace plugins.
- [ ] list page top title, add description: list of enabled plugins.
- [ ] list page, edit button in action area modified to: `配置/Configure`.

## Plugin Editor

- [ ] Some Auth plugins do not need to be configured, in the Data Editor, to disable the input of content.
- [ ] The button is not internationalized.
- [ ] Document button fits too close to JSON button.
- [ ] JSONSchema checks for error messages, which can be divided into: error message + view details.
- [ ] CodeMirror code error highlighting.

## Meta Raw data editor (Route List page)
- [ ] Button not internationalized.
- [ ] layout needs to be adjusted, with additional scrolling axes.

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

  ## Plugin TODO
 * [ ]  Search area, using Table's own, using fuzzy search.
 * [x]  page title not internationalized: configure global plugins/Configure Global Plugin.
 * [ ]  Add icon for Plugin Marketplace plugins.
 * [x]  list page top title, add description: list of enabled plugins.
 * [x]  list page, edit button in action area modified to: `配置/Configure`.
 
 ## Plugin Editor
 * [ ]  Some Auth plugins do not need to be configured, in the Data Editor, to disable the input of content.
 * [x]  The button is not internationalized.
 * [ ]  Document button fits too close to JSON button.
 * [ ]  JSONSchema checks for error messages, which can be divided into: error message + view details.
 * [ ]  CodeMirror code error highlighting.
 
 ## Meta Raw data editor (Route List page)
 * [ ]  Button not internationalized.
 * [ ]  layout needs to be adjusted, with additional scrolling axes.

Extras: Dashboard title also internationalised

**Related issues**

fix/resolve #1631 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_32_08

screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_19_00

screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_33_35

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@nic-chen @starsz Plz take a look at the failed CI.

**Comment by nic-chen** [comment]
> @nic-chen @starsz Plz take a look at the failed CI.

It may be because apisix-docker failed to start.
This problem has been fixed:
https://github.com/apache/apisix-docker/pull/154

I ran again to take a look.

**Comment by iamayushdas** [comment]
@nic-chen @juzhiyuan can i have the review please?

**Comment by juzhiyuan** [comment]
@iamayushdas of course! Everyone could review PRs in our community :)

**Comment by juzhiyuan** [comment]
Just review & comment if needed :)

**Comment by iamayushdas** [comment]
> @iamayushdas of course! Everyone could review PRs in our community :)

I meant ,Can you review this for approval

**Comment by juzhiyuan** [comment]
got it, please be patient for GitHub is not realtime always :)

**Comment by nic-chen** [comment]
> @nic-chen @juzhiyuan @LiteSun @guoqqqi can i have the review please?

sorry, I am not familiar with the front-end codes and can't review it for you

**Review by LiteSun — APPROVED** [review]

**Comment by iamayushdas** [comment]
@liuxiran please review this

File: web/src/locales/zh-CN/component.ts [hunk 1]
```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
```
Comment (juzhiyuan) [inline web/src/locales/zh-CN/component.ts:24]: I remember I have reviewed one PR like this 😳 Why will I see it again?

Comment (iamayushdas) [reply web/src/locales/zh-CN/component.ts:24]: idk i guess there is some commit issue.

Comment (juzhiyuan) [reply web/src/locales/zh-CN/component.ts:24]: haha, let's review one by one  currently.

Comment (iamayushdas) [reply web/src/locales/zh-CN/component.ts:24]: Yes yes

**Comment by juzhiyuan** [comment]
Hi, @LiteSun @Jaycean @liuxiran @bzp2010 Please view this one first, once that gets merged, @iamayushdas Please update this branch to latest.

https://github.com/apache/apisix-dashboard/pull/1685

**Comment by Jaycean** [comment]
@iamayushdas  Please update the branch to resolve the code conflict.

**Review by juzhiyuan — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Comment by iamayushdas** [comment]
@juzhiyuan all approvals are done now, you can merge it

**Pull request merged**